pom.xml

```diff
@@ -52,6 +52,7 @@
     <module>watcher/api</module>
     <module>watcher/standard</module>
     <module>it</module>
+    <module>rt</module>
```

Comment:
> What's the difference between regression and integration test ?

Regression test essentially tests build and deployment, redeployment aspects of f8-m-p. It sounds to have some overlap with integration tests in general for f8m-p.

> Also it might make sense to move this module to the regression-test profile

Agreed. Right now we are its child module because eventually, we want to take regression test module out from f8-m-p. For the time being, we thought f8m-p is the right placeholder for these test modules.